LayoutTests/http/wpt/webcodecs/videoFrame-duration.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<header>
+<script src='/resources/testharness.js'></script>
+<script src='/resources/testharnessreport.js'></script>
+</header>
+<body>
+<script>
+
+function makeOffscreenCanvas(width, height, timestamp, duration) {
```

Comment:
The name “makeOffscreenCanvas” made me think it was creating a canvas element. Maybe “videoFrameFromOffscreenCanvas” or “makeFrameFromOffscreenCanvas”?

Reply:
yep, will fix